libursa/src/ffi/cl/prover.rs

```diff
@@ -945,6 +945,40 @@ pub extern "C" fn ursa_cl_prover_new_proof_builder(
     res
 }
 
+/// Add a common attribute to the proof builder
+///
+/// # Arguments
+/// * `proof_builder` - Reference that contain proof builder instance pointer.
+/// * `attribute_name` - Common attribute's name
+
+#[no_mangle]
+pub extern "C" fn ursa_cl_proof_builder_add_common_attribute(
+    proof_builder: *const c_void,
+    attribute_name: *const c_char,
```

Comment:
We really need to get away from pointers. We can't preserve a lot of Rust guarantees if the API allows arbitrary data injection.

Reply:
Point noted. I was just following the pattern already being followed.